### PR TITLE
Deploy Helm chart with values

### PR DESCRIPTION
**Pull request by johnwatson484**
https://eaflood.atlassian.net/browse/PSD-646

### CODE REVIEW

File: vars/helm.groovy [hunk 1]
```diff
@@ -1,12 +1,41 @@
+def getExtraCommands(chartName, tag) {
```
Comment (st3v3nhunt) [inline vars/helm.groovy:1]: `chartName` doesn't look to be used within this function - can it be removed?

File: vars/helm.groovy [hunk 2]
```diff
@@ -1,12 +1,41 @@
...
+  return "--set labels.version=$tag  --install --atomic"
```
Comment (st3v3nhunt) [inline vars/helm.groovy:2]: There is an extra space after `...$tag` and before `--install`

File: vars/helm.groovy (hunk 1 above)
Comment (johnwatson484) [reply vars/helm.groovy]: Removed

File: vars/helm.groovy (hunk 2 above)
Comment (johnwatson484) [reply vars/helm.groovy]: Good spot, removed!

**Pull request merged**